libs/sdk-react-native/ios/RNBreezSDK.swift

```diff
@@ -414,11 +414,12 @@ class RNBreezSDK: RCTEventEmitter {
         }
     }
 
-    @objc(fetchReverseSwapFees:rejecter:)
-    func fetchReverseSwapFees(_ resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
+    @objc(fetchReverseSwapFees:resolver:rejecter:)
+    func fetchReverseSwapFees(_ reqData:[String: Any], resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
         do {
-            let fees = try getBreezServices().fetchReverseSwapFees()
-            resolve(fees)        
+            let reverseSwapFeesRequest = BreezSDKMapper.asReverseSwapFeesRequest(reqData: reqData)
+            let fees = try getBreezServices().fetchReverseSwapFees(reqData: reverseSwapFeesRequest)
```

Comment:
Should be `req` instead of `reqData`

Reply:
Done in 6771091833e2e23095f676ecb07d7dfe507683b4